src/geometrycollection.h

```diff
@@ -33,6 +33,9 @@ class GeometryCollection : public QAbstractListModel
         /// Remove and unload all geometries whose filenames matched by given QRegExp object
         void unloadFiles(const QRegExp & filenameRegex);
 
+        /// Find QModelIndex matching the label
```

Comment:
Let's be slightly more specific here: `Find the first index to a geometry with label matching the given pattern`